### PR TITLE
Adding support for linux/ppc64le arch in github action for kserve-agent

### PR DESCRIPTION
**Pull request by adilhusain-s**
**What this PR does / why we need it**:
This PR builds and publishes the multi-arch docker image of kserve-agent for linux/ppc64le and linux/amd64 arch.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Type of changes**
Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevent result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Checklist**:

- [ ] Have you added unit/e2e tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by yuzisun** [comment]
@adilhusain-s how do we test and verify the multi-arch image ?

**Comment by adilhusain-s** [comment]
@yuzisun 
please checkout the  workflow [run](https://github.com/adilhusain-s/kserve/actions/runs/3506512371)
you may also pull the docker image using:
docker pull adysh97/agent: latest  # pulls the docker image for current platform/arch 

check the manifest of multi-arch docker image:
 docker manifest inspect adysh97/agent:latest | grep arch -i

**Comment by pranavpandit1** [comment]
@yuzisun - Any Updates on this request, I'd be interested in these changes.

File: .github/workflows/agent-docker-publish.yml [hunk 1]
```diff
@@ -68,9 +76,14 @@ jobs:
...
-          docker tag $IMAGE_NAME $IMAGE_ID:$VERSION
-          docker push $IMAGE_ID:$VERSION
-
+          echo VERSION=$VERSION >> $GITHUB_ENV
+          echo IMAGE_ID=$IMAGE_ID >> $GITHUB_ENV
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          platforms: linux/amd64,linux/ppc64le
```
Comment (yuzisun) [inline .github/workflows/agent-docker-publish.yml]: Update the platforms to match other PRs

**Comment by adilhusain-s** [comment]
@yuzisun 
could you review and merge this PR?

**Pull request merged**